### ISSUE TITLE
[Feature Request] Bring in Basic Accessibility testing code scans

### ISSUE THREAD

**Issue by jsdev** [vuetifyjs/vuetify#17846]
### Problem to solve
I unfortunately can't use vuetify as is or recommend any teams in my program to use, but teams like vue. Vuetify 3 took accessibility compliance step backwards, because there is no accessibility mark up scanning. Currently screen readers can't read many form fields because nesting inputs in role="textbox" and no label associated with parent, also selects problematic and no native select option is available. in the meantime.

### Proposed solution
incorporate accessibility code scans, to get a feedback loop of the current violations and discourage new violations or regression going forward.

**Comment by websitevirtuoso** [comment]
Could you please assist in this question?

How to implement this feature and which tools use for testing? Maybe somebody could be interested to help and do PR.
Hope on your assistance
